Rocket.Chat/Views/Cells/Chat/ChatMessageUnreadSeparator.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ChatMessageUnreadSeparator.swift
+//  Rocket.Chat
+//
+//  Created by Rafael Kellermann Streit on 19/12/16.
+//  Copyright © 2016 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+final class ChatMessageUnreadSeparator: UICollectionViewCell {
+    static var minimumHeight: CGFloat = 40
+
+    static let identifier = "ChatMessageDaySeparator"
```

Comment:
Shouldn't this be "ChatMessageUnreadSeparator"?

Reply:
Oops! Nice catch @soundstorm !